examples/firebase-auth-firestore/package.json

```
@@ -26,6 +29,6 @@
     "typescript": "^4.6.3"
   },
   "engines": {
-    "node": ">=14"
+    "node": "16"
```

is there a reason why we put this on v16 here?

`@google-cloud/functions-framework` works with `>=10.0.0` & `firebase-functions` works with `^8.13.0 || >=10.10.0`
https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/aa58c8789c0e03619ba470681e42bb44d825e867/package.json#L5-L7
https://github.com/firebase/firebase-functions/blob/40f0b4331517755fcd0a906d9e5361ad7305aa05/package.json#L230-L232

```suggestion
    "node": ">=14"
```

---

Let's keep this at `14`, as that's the oldest possible version.

```suggestion
    "node": "14"
```

---

This sets the environment we are deploying to and doesn't add any peer requirements, as nothing should be using `examples/firebase-auth-firestore` as a dependency.

I can only see setting the target architecture to 14 as being a disadvantage to users, as Node 14 is in Maintenance LTS with EOL 2023-04-30, whereas Node 16 is in Active LTS with EOL 2024-04-30 ([source](https://nodejs.org/en/about/releases/)).

Unless you are thinking Remix works better or is better tested on Node 14? Or users are more likely to encounter other compatibility issues if they deploy to 16?

Or is there another advantage to setting this to 14 that I've missed?

---

We currently have all our examples set to Node v14 (because that's the minimum possible version we support), so it would be better to keep this consistent.